maintainers/team-list.nix

```diff
@@ -255,7 +472,54 @@ with lib.maintainers; {
       vdemeester
       zowoq
     ];
+    githubTeams = [
+      "podman"
+    ];
     scope = "Maintain Podman and CRI-O related packages and modules.";
+    shortName = "Podman";
+    enableFeatureFreezePing = true;
+  };
+
+  postgres = {
+    members = [
+      thoughtpolice
+    ];
+    scope = "Team for feature-freeze-teams.pl, not to be used as package maintainers!";
+    shortName = "PostgreSQL";
+    enableFeatureFreezePing = true;
+  };
+
+  python = {
+    members = [
+      fridh
+      hexa
+      jonringer
+    ];
+    scope = "Team for feature-freeze-teams.pl, not to be used as package maintainers!";
+    shortName = "Python";
+    enableFeatureFreezePing = true;
+  };
+
+  qt-kde = {
+    members = [
+      ttuegel
+    ];
+    githubTeams = [
+      "qt-kde"
+    ];
+    scope = "Team for feature-freeze-teams.pl, not to be used as package maintainers!";
```

Comment:
I do not see why this should not be used for package maintainers if Thomas wanted to do so. I would suspect most people who have not yet switched to teams for their package collections are just too busy to carry that out.

Reply:
I had similar concerns about the wording. https://github.com/NixOS/nixpkgs/pull/165978#discussion_r836402127

Reply:
Do you have an alternative wording? Because this is the best that springs out of my tired brain